runner/skeptic_gate_cli.py

```
@@ -88,6 +89,95 @@
+            check=False,
+        ).stdout.decode("utf-8", errors="replace")
+    except Exception as exc:
+        return f"(could not enumerate changed files: {exc})"
+    lines = [line for line in status.splitlines() if line.strip()]
+    return "; ".join(lines[:10]) if lines else "(no porcelain status -- binary/ignored-file change?)"
+
+
+def _restore_checkout(repo: pathlib.Path) -> None:
+    """Discard any mutation a compromised reviewer subprocess left behind."""
+    subprocess.run(
+        ["git", "-C", str(repo), "checkout", "--quiet", "--", "."],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)